catalyst/data/sampler.py

```diff
@@ -1,8 +1,13 @@
-from typing import Iterator, List  # isort:skip
+from operator import itemgetter
+from typing import Iterator, List
+
 import numpy as np
 
+from torch.utils.data import DistributedSampler
 from torch.utils.data.sampler import Sampler
 
+from catalyst.data.dataset import DatasetFromSampler
```

Comment:
could you please replace it with `from catalyst.data import DatasetFromSampler`?

Reply:
done